src/System.Security.Cryptography.Cng/ref/project.json

```diff
@@ -1,8 +1,8 @@
 {
   "dependencies": {
-    "System.Runtime": "4.0.0",
-    "System.IO": "4.0.0",
-    "System.Runtime.Handles": "4.0.0",
+    "System.Runtime": "4.1.0-rc3-24117-00",
```

Comment:
Yeah, this LGTM.  All of these would have shown up as downgrades from `dotnet restore` as well, but I don't think we're propogating any of those warnings...